packages/salesforcedx-vscode-core/src/commands/forceAuthDevHub.ts

```diff
@@ -37,6 +48,49 @@ export class ForceAuthDevHubExecutor extends SfdxCommandletExecutor<{}> {
     command.withArg('--setdefaultdevhubusername');
     return command.build();
   }
+
+  public async execute(response: ContinueResponse<any>): Promise<void> {
+    const cancellationTokenSource = new vscode.CancellationTokenSource();
+    const cancellationToken = cancellationTokenSource.token;
+
+    const execution = new CliCommandExecutor(this.build(response.data), {
+      cwd: getRootWorkspacePath()
+    }).execute(cancellationToken);
+
+    execution.processExitSubject.subscribe(async () => {
+      const globalDevHubName = await OrgAuthInfo.getDefaultDevHubUsernameOrAlias(
+        false,
+        ConfigSource.Global
+      );
+
+      if (isNullOrUndefined(globalDevHubName)) {
+        const localDevHubName = await OrgAuthInfo.getDefaultDevHubUsernameOrAlias(
+          false,
+          ConfigSource.Local
+        );
+
+        if (isNullOrUndefined(localDevHubName) === false) {
```

Comment:
Can we make this `if (localDevHubName)`? A bit more readable than the double negative.